lib/dsl/poc/ast_builder.js

```diff
@@ -102,6 +100,90 @@ class JDLAstBuilderVisitor extends BaseJDLCSTVisitor {
       pattern: ctx.REGEX[0].image
     };
   }
+
+  relationDecl(ctx) {
+    // TODO: implement
+  }
+
+  relationshipType(ctx) {
+    // TODO: implement
+  }
+
+  relationshipBody(ctx) {
+    // TODO: implement
+  }
+
+  relationshipSide(ctx) {
+    // TODO: implement
+  }
+
+  enumDecl(ctx) {
+    // TODO: implement
+  }
+
+  enumPropList(ctx) {
+    // TODO: implement
+  }
+
+  dtoDecl(ctx) {
+    // TODO: implement
+  }
+
+  entityList(ctx) {
+    // TODO: implement
+  }
+
+  method(ctx) {
+    // TODO: implement
+  }
+
+  exclusion(ctx) {
+    // TODO: implement
+  }
+
+  paginationDecl(ctx) {
+    // TODO: implement
+  }
+
+  serviceDecl(ctx) {
+    // TODO: implement
+  }
+
+  microserviceDecl1(ctx) {
+    // TODO: implement
+  }
+
+  searchEngineDecl(ctx) {
+    // TODO: implement
+  }
+
+  noClientDecl(ctx) {
+    // TODO: implement
+  }
+
+  noServerDecl(ctx) {
+    // TODO: implement
+  }
+
+  noFluentMethod(ctx) {
+    // TODO: implement
+  }
+
+  filterDecl(ctx) {
+    // TODO: implement
+  }
+
+  negativeForDecl(ctx) {
```

Comment:
What is it?

Reply:
This is the one I need to rename to filterDef
https://github.com/jhipster/jhipster-core/blob/master/lib/dsl/poc/parser.js#L293